ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -26,3 +26,14 @@ def swapdims(input, dim0, dim1):
     "uint32",
     "uint64",
 )
+
+
+def transpose(input, dim0, dim1):
+    return ivy.swapaxes(input, dim0, dim1)
+
+
+transpose.unsupported_dtypes = (
```

Comment:
Are you sure that uint dtypes cannot be used for this function?

Reply:
according to ivy.swapaxes, the uint16, uint32, uint64 does seem to be unsupported

Reply:
Are you sure? I've just ran it locally and it is accepting arrays with uint dtypes.

Reply:
This is the definition from `(https://github.com/unifyai/ivy/tree/master/ivy/functional/backends/torch)/manipulation.py`
it says `swapaxes.unsupported_dtypes` doesn't support uint types; did I misunderstood this or

```
def swapaxes(
    x: torch.Tensor, axis0: int, axis1: int, *, out: Optional[torch.Tensor] = None
) -> torch.Tensor:
    ret = torch.transpose(x, axis0, axis1)
    return ret


swapaxes.unsupported_dtypes = (
    "uint16",
    "uint32",
    "uint64",
)
```

Reply:
Yep that is the backend PyTorch implementation of `swapaxes`. However, the frontend implementation is independent of this and `ivy.swapaxes()` has been designed as to accept any relevant dtype regardless of the framework. However, I have raised this with the team and will get back with an answer!

Reply:
ok got it. so what's our policy here regarding the `frontend_function.unsupported_dtypes = ()`; should I keep including them in the frontend or not include them?